src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Dataplane/Azure.AnalysisServices.psd1

```diff
@@ -48,7 +48,7 @@ CLRVersion='4.0'
 ProcessorArchitecture = 'None'  
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '4.6.0'; })
+RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '4.6.1'; })
```

Comment:
Missed the bump to 5.0.0 here for Profile - could you bump this in the next commit?